captum/attr/_core/lrp.py

```diff
@@ -151,11 +190,11 @@ def attribute(
         """
         self.verbose = verbose
         self._original_state_dict = self.model.state_dict()
-        self.layers = []
+        self.layers: List[Any] = []
         self._get_layers(self.model)
         self._check_and_attach_rules()
-        self.backward_handles = []
-        self.forward_handles = []
+        self.backward_handles: List[Any] = []
+        self.forward_handles: List[Any] = []
```

Comment:
Can this can be more precise as List[RemovableHandle] similar to this?
https://github.com/pytorch/captum/blob/11cc1e64889fbb2802a04224a1df2013bcaa7072/captum/attr/_core/deep_lift.py#L131